apps/judicial-system/backend/migrations/20231120164318-update-case.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface) {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.sequelize
+        .query(
+          'SELECT id, court_documents FROM "case" WHERE court_documents IS NOT NULL',
+          { transaction: t },
+        )
+        .then((res) =>
+          Promise.all(
+            res[0].map((c) => {
+              const fix = c.court_documents.reduce(
+                (acc, d, i) =>
+                  acc +
+                  (i === 0 ? '"' : ',"') +
+                  JSON.stringify({
+                    name: d.name,
+                    submittedBy:
+                      d.submittedBy === 'JUDGE'
+                        ? 'DISTRICT_COURT_JUDGE'
+                        : d.submittedBy,
+                  })
+                    .replace(/\\/g, '\\\\')
+                    .replace(/"/g, '\\"')
+                    .replace(/'/g, "''") +
+                  '"',
+                '',
+              )
+              console.log(c.court_documents, fix)
```

Comment:
Did you leave the console.log in on purpose?

Reply:
🫣